libmscore/chordrest.cpp

```diff
@@ -562,9 +562,11 @@ Element* ChordRest::drop(EditData& data)
                         ic->setTrack((track() / VOICES) * VOICES);
                         Instrument* instr = ic->instrument();
                         Instrument* prevInstr = part()->instrument(tick());
+                        // Only setup instrument change if changing to a different instrument.
                         if (instr && instr->isDifferentInstrument(*prevInstr)) {
                               ic->setupInstrument(instr);
                               }
+                        InstrumentChange* nextIc = score()->nextInstrumentChange(segment(), part(), false);
```

Comment:
This variable isn't used. What reason is there for adding this line?

compiler warning:
```
libmscore/chordrest.cpp:569:43: warning: unused variable ‘nextIc’ [-Wunused-variable]
                         InstrumentChange* nextIc = score()->nextInstrumentChange(segment(), part(), false);
```

Reply:
Oh, that was from when I was trying something out, that didn't work. I've removed that now.